### PR TITLE
SAK-31664 update velocity framework to use SLF4JLogChute

### PR DESCRIPTION
**Pull request by ern**

### CODE REVIEW

File: rights/rights-tool/tool/src/webapp/velocity.properties [hunk 1]
```diff
@@ -8,13 +8,7 @@
 #----------------------------------------------------------------------------
 
 # log to our existing log4j
```
Comment (buckett) [inline rights/rights-tool/tool/src/webapp/velocity.properties]: All these comments should be corrected now.

**Comment by buckett** [comment]
Typo in commit message (frameowrk).

At the moment logging is effectively disabaled in all the velocity tools by lots of:

    runtime.log.logsystem.log4j.category = vm.none		

this change looks to enable logging in all the velocity templating and have it all log under the same logger. Previously if I was working on a tool I would just go into the _velocity.properties_ and turn on logging for just that tool while I was working on it. Is this still easy in the new setup?

From looking I'd have to add this to the _velocity.properties_ file:

    runtime.log.logsystem.slf4j.logger=org.sakaiproject.sitemanage.velocity

and then configure the SLF4J logging to ignore all the standard logging on `org.sakaiproject.velocity.util.SLF4JLogChute`and enable debug logging on `org.sakaiproject.sitemanage.velocity`

I want all the logging to go to SLF4J, but it seems like this isn't quite there. Could we have an easy way to enable/disable the chute so that by default they can all be disabled but when working on a tool we can just enable one and have this all done through the velocity.properties.

**Comment by ern** [comment]
@buckett Thanks for the suggestion reimplemented see the JIRA for the details https://jira.sakaiproject.org/browse/SAK-31664

**Comment by ern** [comment]
yeah agree when sakai upgrades to velocity 2 it will be an easy removal!

**Pull request merged**